experiments/speedrun/llama_75m_fineweb_edu/llama_75m_fineweb_edu.py

```diff
@@ -8,26 +8,29 @@
 from experiments.simple_train_config import SimpleTrainConfig
 from marin.execution.executor import executor_main
 from marin.resources import TpuPodConfig
-from marin.speedrun.speedrun import HardwareConfig, SpeedrunConfig, default_speedrun
+from marin.speedrun.speedrun import Author, SpeedrunConfig, default_speedrun
 
 logger = logging.getLogger("ray")
 
 speedrun_config = SpeedrunConfig(
+    author=Author(
+        name="Nikil Ravi",
+        affiliation="Marin Community",
+        url="https://www.linkedin.com/in/nikilravi/",
+    ),
+    description="75M parameter model based on LLaMA architecture.",
```

Comment:
```suggestion
    description="75M parameter model based on Llama architecture.",
```

(LLaMA was the first one)